### ISSUE TITLE
Optionally add prefix or show dialog (Save as or warning) only if target file exists in folder?

### ISSUE THREAD

**Issue by puyebu** [Dezaimasu/cute-button#33]
Frequently in cases when I save images from discord or some other sources they are named as "unknown.png" or have some other same filename despite being different and sometimes I resave the save image from some image board finding it from another place.

As of now it blinks red and downloads while automatically adding numeric postfix, encountering the same filename.

I rememer there being option to display "Save as" dialog if file exists but I don't see the option in the current version.

Ideally, there would be an option to add prefix or show "Save as" dialog or show warning (file exists without starting download) tooltip at first if file exists for specific hosts.

Basically what I am talking about is making 
`conflictAction: 'uniquify',` 
configurable from settings panel on per host basis.

**Comment by Dezaimasu** [comment]
_more necroposting_ 

![image](https://github.com/Dezaimasu/cute-button/assets/1636242/57bb6757-e932-4b98-9c25-d659b76ad358)
It's still there. Configurable conflict action is easy, but turned out Firefox is not supporting "prompt" option, so the whole thing is useless.
